docs/source/explanation/algorithms/segmentation/semantic_segmentation.rst

```diff
@@ -1,23 +1,145 @@
+
+.. code-block::
+
+    $ otx {train, optimize} <model_template> --train-data-root <path_to_train_images_folder> --val-data-root <path_to_val_images_folder> --train-ann-files <path_to_train_segmentation_masks_folder> --val-ann-files <path_to_val_segmentation_masks_folder>
+
+.. note::
+
+    Please, refer to our :doc:`dedicated tutorial <../../../tutorials/base/how_to_train/semantic_segmentation>` for more information on how to train, validate and optimize the semantic segmentation model.
 
 ******
 Models
 ******
 
-To be added soon
+We support the following ready-to-use model templates:
+
++------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------+---------------------+-----------------+
+| Template ID                                                                                                                                                                                                                  | Name                   | Complexity (GFLOPs) | Model size (MB) |
++==============================================================================================================================================================================================================================+========================+=====================+=================+
+| `Custom_Semantic_Segmentation_Lite-HRNet-s-mod2_OCR <https://github.com/openvinotoolkit/training_extensions/blob/feature/otx/otx/algorithms/segmentation/configs/ocr_lite_hrnet_s_mod2/template.yaml>`_                      | Lite-HRNet-s-mod2      | 1.82                | 3.5             |
++------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------+---------------------+-----------------+
+| `Custom_Semantic_Segmentation_Lite-HRNet-18_OCR <https://github.com/openvinotoolkit/training_extensions/blob/feature/otx/otx/algorithms/segmentation/configs/ocr_lite_hrnet_18/template.yaml>`_                              | Lite-HRNet-18          | 3.45                | 4.5             |
++------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------+---------------------+-----------------+
+| `Custom_Semantic_Segmentation_Lite-HRNet-18-mod2_OCR <https://github.com/openvinotoolkit/training_extensions/blob/feature/otx/otx/algorithms/segmentation/configs/ocr_lite_hrnet_18_mod2/template.yaml>`_                    | Lite-HRNet-18-mod2     | 3.63                | 4.8             |
```

Comment:
But we have two different templates. I see the solution simply not mention the first version of the Lite-HRNet-18. I think it will only misslead a user

Reply:
Oh yes, I agree! My concern will be resolved absolutely after updating complexity and model size for each templates. Let's keep one row for one template.